packages/jest-dom-mocks/src/test/document.test.ts

```diff
@@ -0,0 +1,40 @@
+import Document from '../document';
+
+describe('Document', () => {
+  describe('mock', () => {
```

Comment:
Can we test that when it's `mocked` calling `addEventListener` and `removeEventListener` doesn't do anything / that you can assert on them?